### DISCUSSION TITLE
Fatal error: Uncaught Error: Class "FPDF" not found in ..InvoicePrinter.php:18

### DISCUSSION THREAD

**Discussion by unluckychild** [artkonekt/pdf-invoice#52] [Q&A] [answered]
composer install and opened examples. This shows up. What now? I used similar, maybe same version here
https://github.com/farjadtahir/pdf-invoicr without problem

**Comment by ay4t** [comment, marked as answer]
FPDF not found I think you haven't installed FPDF? in composer.json requires   ​"setasign/fpdf"​: ​"​~1.8.1​"

**Reply by unluckychild** [reply]
Im sorry, im totally new with this composers. I just typed composer install. How do i install pfdf please?

**Reply by unluckychild** [reply]
I tried "composer require setasign/fpdf" but shows nothing to update or install

**Reply by unluckychild** [reply]
LOL, i found a solution:
Just added
require '../vendor/autoload.php'; before scripts and its good now
